tests/app/questionnaire/test_placeholder_parser.py

```diff
@@ -95,6 +97,39 @@ def test_metadata_transform_placeholder():
         answer_store=AnswerStore(),
         list_store=ListStore(),
         metadata={"ref_p_start_date": "2019-02-11"},
+        response_metadata={},
+        schema=QuestionnaireSchema({}),
+    )
+    placeholders = parser(placeholder_list)
+
+    assert placeholders["start_date"] == "Monday 11 February 2019"
+
+
+def test_response_metadata_transform_placeholder():
```

Comment:
Caveat that this is a point in time test as we don't support ISO format dates yet.

Reply:
Added a comment in that test for future reference.